### ISSUE TITLE
After Eviction, the application uses default-scheduler, causing it to fail to run normally.

### ISSUE THREAD

**Issue by FloatXD** [hwameistor/hwameistor#1315]
Evicte node1

```
[root@k8s-master ~]# kubectl describe  pod nginx-local-storage-lvm95-85df588459-wcqrr
Name:             nginx-local-storage-lvm95-85df588459-wcqrr
Namespace:        default
Priority:         0
Service Account:  default
Node:             k8s-master/10.6.112.50
Start Time:       Mon, 11 Dec 2023 16:26:44 +0800
Labels:           app=nginx-local-storage-lvm95
                  pod-template-hash=85df588459
Annotations:      <none>
Status:           Pending
IP:               
IPs:              <none>
Controlled By:    ReplicaSet/nginx-local-storage-lvm95-85df588459
Containers:
  nginx:
    Container ID:  
    Image:         nginx:latest
    Image ID:      
    Port:          80/TCP
    Host Port:     0/TCP
    Command:
      sh
      -xc
      VOL="$( df | grep /usr/share/nginx/html | awk '{print $1,$NF}' )"
      echo "<center><h1>Demo volume: ${VOL}</h1></center>" > /usr/share/nginx/html/index.html
      nginx -g "daemon off;"
      
    State:          Waiting
      Reason:       ContainerCreating
    Ready:          False
    Restart Count:  0
    Limits:
      cpu:     20m
      memory:  30Mi
    Requests:
      cpu:        20m
      memory:     30Mi
    Environment:  <none>
    Mounts:
      /usr/share/nginx/html from html-root (rw)
      /var/run/secrets/kubernetes.io/serviceaccount from kube-api-access-qjrg8 (ro)
Conditions:
  Type              Status
  Initialized       True 
  Ready             False 
  ContainersReady   False 
  PodScheduled      True 
Volumes:
  html-root:
    Type:       PersistentVolumeClaim (a reference to a PersistentVolumeClaim in the same namespace)
    ClaimName:  local-storage-pvc-lvm95
    ReadOnly:   false
  kube-api-access-qjrg8:
    Type:                    Projected (a volume that contains injected data from multiple sources)
    TokenExpirationSeconds:  3607
    ConfigMapName:           kube-root-ca.crt
    ConfigMapOptional:       <nil>
    DownwardAPI:             true
QoS Class:                   Guaranteed
Node-Selectors:              <none>
Tolerations:                 node.kubernetes.io/not-ready:NoExecute op=Exists for 300s
                             node.kubernetes.io/unreachable:NoExecute op=Exists for 300s
Events:
  Type     Reason              Age                 From                     Message
  ----     ------              ----                ----                     -------
  Normal   Scheduled           13m                 default-scheduler        Successfully assigned default/nginx-local-storage-lvm95-85df588459-wcqrr to k8s-master
  Warning  FailedAttachVolume  13m                 attachdetach-controller  Multi-Attach error for volume "pvc-cd39c858-b56d-4550-926e-1e06846cb017" Volume is already exclusively attached to one node and can't be attached to another
  Warning  FailedAttachVolume  28s (x14 over 12m)  attachdetach-controller  AttachVolume.Attach failed for volume "pvc-cd39c858-b56d-4550-926e-1e06846cb017" : rpc error: code = Unknown desc = not found valid volume replica
```

```
[root@k8s-master ~]# kubectl get pod -n hwameistor -o wide
NAME                                                       READY   STATUS             RESTARTS        AGE    IP                NODE         NOMINATED NODE   READINESS GATES
hwameistor-admission-controller-d684c7b75-9bjwc            1/1     Running            0               14m    100.111.156.121   k8s-node1    <none>           <none>
hwameistor-admission-controller-d684c7b75-f8sm9            1/1     Terminating        2 (111m ago)    3d6h   100.111.156.83    k8s-node1    <none>           <none>
hwameistor-admission-controller-d684c7b75-fn4tw            1/1     Running            0               14m    100.111.156.69    k8s-node1    <none>           <none>
hwameistor-admission-controller-d684c7b75-k95rf            1/1     Running            0               75m    100.116.59.87     k8s-master   <none>           <none>
hwameistor-apiserver-6f6579b5c-kkrt7                       1/1     Terminating        1 (113m ago)    3d6h   100.64.169.142    k8s-node2    <none>           <none>
hwameistor-apiserver-6f6579b5c-v872c                       1/1     Running            0               14m    100.111.156.114   k8s-node1    <none>           <none>
hwameistor-auditor-6778bb5df7-djx62                        0/1     CrashLoopBackOff   7 (3m30s ago)   14m    100.111.156.112   k8s-node1    <none>           <none>
hwameistor-exporter-75cc84967-rrcks                        1/1     Running            0               14m    100.111.156.118   k8s-node1    <none>           <none>
hwameistor-failover-assistant-6c675d589b-m5s7l             1/1     Running            0               14m    100.111.156.94    k8s-node1    <none>           <none>
hwameistor-local-disk-action-controller-6878df7545-2cl5k   1/1     Running            0               14m    100.111.156.113   k8s-node1    <none>           <none>
hwameistor-local-disk-csi-controller-6f6fc5875b-jqwr9      2/2     Terminating        2 (113m ago)    3d6h   100.64.169.144    k8s-node2    <none>           <none>
hwameistor-local-disk-csi-controller-6f6fc5875b-rd7q9      2/2     Running            0               14m    100.116.59.118    k8s-master   <none>           <none>
hwameistor-local-disk-manager-b2f9j                        2/2     Running            2 (113m ago)    3d6h   10.6.112.50       k8s-master   <none>           <none>
hwameistor-local-disk-manager-q4972                        2/2     Running            2 (113m ago)    3d6h   10.6.112.51       k8s-node1    <none>           <none>
hwameistor-local-disk-manager-tr7dk                        2/2     Running            2 (113m ago)    3d6h   10.6.112.52       k8s-node2    <none>           <none>
hwameistor-local-storage-c4f5w                             2/2     Running            2 (113m ago)    3d6h   100.116.59.82     k8s-master   <none>           <none>
hwameistor-local-storage-csi-controller-69dd5654dc-tj8jp   6/6     Running            6 (113m ago)    3d6h   100.116.59.85     k8s-master   <none>           <none>
hwameistor-local-storage-l6bs8                             2/2     Running            3 (111m ago)    3d6h   100.111.156.82    k8s-node1    <none>           <none>
hwameistor-local-storage-vkf4s                             2/2     Running            2 (113m ago)    3d6h   100.64.169.145    k8s-node2    <none>           <none>
hwameistor-pvc-autoresizer-648f4d7bbd-b96l7                1/1     Running            0               14m    100.111.156.119   k8s-node1    <none>           <none>
hwameistor-scheduler-58cb757788-46r56                      1/1     Running            0               74m    100.116.59.88     k8s-master   <none>           <none>
hwameistor-scheduler-58cb757788-7tqpd                      1/1     Terminating        1 (113m ago)    3d6h   100.111.156.85    k8s-node1    <none>           <none>
hwameistor-scheduler-58cb757788-vxgtf                      1/1     Running            0               14m    100.111.156.90    k8s-node1    <none>           <none>
hwameistor-scheduler-58cb757788-zhhcs                      1/1     Running            0               14m    100.111.156.80    k8s-node1    <none>           <none>
hwameistor-ui-78756c7557-9qn9d                             1/1     Terminating        1 (113m ago)    3d6h   100.64.169.143    k8s-node2    <none>           <none>
hwameistor-ui-78756c7557-cxkfv                             1/1     Running            0               14m    100.111.156.76    k8s-node1    <none>           <none>
hwameistor-volume-evictor-569cbf4bcd-vzkqm                 1/1     Running            0               14m    100.111.156.116   k8s-node1    <none>           <none>
hwameistor-volume-evictor-569cbf4bcd-ztm6b                 1/1     Terminating        1 (113m ago)    3d6h   100.111.156.84    k8s-node1    <none>           <none>
```

**Comment by buffalo1024** [comment]
In this case, you can set the hwameistor-admission-controller to ha to ensure pods using hwameistor volumes will be patched to be scheduled by hwameistor-scheduler.

**Comment by FloatXD** [comment]
fix in hwameistor v0.14.3
